flatpak/use-system-lib-flac.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/makefile b/makefile
+index c5cedbc0..797ea027 100644
+--- a/makefile
++++ b/makefile
+@@ -66,7 +66,7 @@
+ # USE_SYSTEM_LIB_EXPAT = 1
+ # USE_SYSTEM_LIB_ZLIB = 1
+ # USE_SYSTEM_LIB_JPEG = 1
+-# USE_SYSTEM_LIB_FLAC = 1
++USE_SYSTEM_LIB_FLAC = 1
+ # USE_SYSTEM_LIB_LUA = 1
+ # USE_SYSTEM_LIB_SQLITE3 = 1
+ # USE_SYSTEM_LIB_PORTMIDI = 1
```

Comment:
As with the other makefile patch, this should really be done in the make command.